<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#Widget{

background:rgb(255, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;

}
QFrame{
border:sold 10px rgba(0,0,0);
	border-image: url(:/new/prefix1/2.231.jpg);
}
QLabel{
border-image:rgba(85,170,25);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QPushButton:pressed{
background-color: rgb(255, 85, 0);
border-style:inset;
font-style:MingLiU-ExtB;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>330</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#8d98a1;
background-color:rgba(255, 255, 255, 35%);
font-size:20px;
border-style:outset;
border-radius:14px;
font-style:MingLiU-ExtB;

color: rgb(255, 255, 255);
font: 16pt &quot;黑体&quot;;</string>
    </property>
    <property name="placeholderText">
     <string> 用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#8d98a1;
background-color:rgba(255, 255, 255, 35%);
font-size:20px;
border-style:outset;
border-radius:14px;
font-style:MingLiU-ExtB;

color: rgb(255, 255, 255);
font: 16pt &quot;黑体&quot;;</string>
    </property>
    <property name="placeholderText">
     <string> 密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;黑体&quot;;
</string>
    </property>
    <property name="text">
     <string>西北工业大学</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>510</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 27pt &quot;黑体&quot;;
color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Myungjo Std M</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Myungjo Std M</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{

	background-color: rgb(255, 5, 9,80%);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background-color: rgb(176, 184, 183,65%);
border-radius:20px;
}
QPushButton{

font: 22pt &quot;黑体&quot;;
background-color: rgb(119, 150, 179);


}
QPushButton:pressed{
background-color: rgb(170, 219, 226);
border-style:inset;
font-style:MingLiU-ExtB;

}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>提  示</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>用户名或密码错误</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>确 定</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>750</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Time loading...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>421</width>
      <height>326</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{background-color: rgb(176, 184, 183,65%);

border-radius:20px;
}
QPushButton{

font: 40pt &quot;黑体&quot;;

background-color: rgb(54, 54, 54,50%);


}
QPushButton:pressed{
background-color: rgb(170, 219, 226);
border-style:inset;
font-style:MingLiU-ExtB;

}</string>
    </property>
    <widget class="QPushButton" name="pushButton_01">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);

border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_02">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_03">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_00">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_04">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_07">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>220</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_05">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>120</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_06">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>120</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>120</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_08">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>220</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_09">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>220</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>220</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 94, 94);
font: 30pt &quot;黑体&quot;;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>380</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_3{background-color: rgb(176, 184, 183,65%);
border-radius:20px;
}
</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>数据库连接成功</string>
     </property>
    </widget>
    <widget class="QLabel" name="sqlok">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>380</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_4{background-color: rgb(176, 184, 183,65%);
border-radius:20px;
}
</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>数据库连接失败</string>
     </property>
    </widget>
    <widget class="QLabel" name="sqlok_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>username</zorder>
   <zorder>password</zorder>
   <zorder>pushButton</zorder>
   <zorder>toolButton</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_4</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
